guides/common/assembly_configuring-puppet-in-satellite.adoc

```diff
@@ -0,0 +1,27 @@
+ifdef::context[:parent-context-of-assembly_configuring-puppet-in-satellite: {context}]
+
+ifndef::context[]
+[id="assembly_configuring-puppet-in-satellite"]
+endif::[]
+ifdef::context[]
+[id="assembly_configuring-puppet-in-satellite_{context}"]
+endif::[]
+= Configuring Puppet in {Project}
+
+:context: assembly_configuring-puppet-in-satellite
+
+[role="_abstract"]
+Puppet is an optional component in {ProjectX}.
+If you installed {Project} afresh and you want to continue using Puppet, you have to enable it in your {Project}.
+If you upgraded your {Project} and you want to stop using Puppet, you can disable it and purge Puppet data and content, respectively.
+
+include::modules/proc_enabling-puppet.adoc[leveloffset=+1]
+
+ifdef::satellite[]
+[role="_additional-resources"]
```

Comment:
```suggestion
[role="configuring-puppet-additional-resources"]
```

Reply:
I'm not sure that this suggestion is a good idea because of our internal build system compliance. I'll do my best to check.

Reply:
Can you explain [role=""] to me? I don't think I've ever used it.

Reply:
Some headings have standardized anchors (such as this one), which, I guess, are used for some metadata-related purpose. I'm still a bit fuzzy about the purpose, though.
I only know that this role in particular is standardized by the [assembly template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc), which is mentioned in [RH Modularization Guidelines](https://redhat-documentation.github.io/modular-docs/#_additional_resources_4).
That's as much as I can tell you about it at this point.

Reply:
We use the `newdoc` tool ([GH:redhat-documentation/newdoc](https://github.com/redhat-documentation/newdoc)) to generate stubs using those standard templates (assembly, concept, procedure, or reference), which suggest this naming convention.